tools/coredump/testsources/python/gdb-dump-offsets.py

```diff
@@ -0,0 +1,77 @@
+"""
+gdb python script for dumping Python offsets.
```

Comment:
Nice! Thank you for this! This reminds me that we really should make this non-code. I created #191 to track this. Seems Python is the simplest place to start this when the time comes. But definitely for a later PR.